Simulator/Core/Model.cpp

```diff
@@ -68,17 +68,17 @@ void Model::createAllVertices() {
 }
 
 /// Sets up the Simulation.
-void Model::setupSim() {
+void Model::allocateMemory() {
```

Comment:
Seems that this does more than call memory allocation methods. At least, at the end, it creates the index map. That might be just a memory allocation method, however.

Reply:
Yeah, you're right. There's some inconsistent usage of the setup methods, probably why they weren't called allocateMemory to begin with.
Should I undo all the renaming, or just keep the changes for setupEdges and setupVertices?